notebooks/tutorial/8 - Getting tomes from the ADX.ipynb

```diff
@@ -0,0 +1,230 @@
vp4NDIU/TFhu4iFyahY+MSWHj5dmfvx0Q/xFX3N9MrK4ooMbp/qL57+klkiX/8iXj6d+j5r50PylC7eM1aW+1H7lJ7O/TgSioz8sKDvSvdUW0eejHx3eJIm2AAAAACAYiFwBLZDuQWO6+v3F0Tg+H8uzouZ5DfvqDqP2vSHd7R10pEW8bq7X8+xVm6ISpGiofR//E+RVF74n0mx2f2Ff9uT2VBvRv2fPeL1wkO5WfJh4kNzhX982PWfYjPtcEf/cXbP/J/fM9/691vyndR/XjNqX2qTfpQXVYTAUbN472NRz7F1UAZ2K5HuJrPy49tXIqKaoX3he9/8LNf84VbracvCOb2iYsPnf/9aLtdWi3/zfmZ9/fXb5842GJuc84w+NZZnJrGyXuHxg9txcZj1+OAnIhJV05mPB+XR19eHr2iz7o8vvp/ZYeOVqDWQBAAAAACgUAgcge1QRoHjo8XVpf9YCciwT0/0Zv67eP3fbqXS6/+c+e8iT+y5v7b+MPFHbf2WZZE46tu+s7SkvTQDx9Q3/6e24fw3U6Ly4/z3K/+R0la1BI6p5W6ZNnZ/k1yaWur+f4vXPfe1/cnAUTvif//PmcXUox5Z11JGn3q9y//2P5PaDrWTtNd8fF7FCRzXZ746qy05dm10fT0duihSvJ6RxfX1xcjnZ44cff/rJ+sL35zXVvjgmylR8fDJDz+KPh/Tw1dE/4yNn3oik3PxyWjk8aIKCo+ebmzt6L7S/eXQXG7geOzi3ZmVxfgjz3snxIFuPV5/+iR653NxAkdar0WGQsPazq2B42PPu9rrE+e/fDRnbCVOSSMDx8Z3r9z78cnizGCH6PbxRDfdPgIAAAAAioHAEdgOZRM4WqYLPXp9Q5Ub9vYvR7RJxn8yT/zP//G//ePsHhEdLuk54DerYjdm4KhqQf7bO4n732eGpbYEjg9lJcr/S8SUGtWIW4w2oweOxsAyel1LWfUy3S9f/+/zvd+s6FttQZECRz0KbLgSXV+Pdp9oPPJOx9dDochQSM8Bu0fWn37b0aCt0PTRF4M/LKiN5JpHP7pn7WfRXjNRkxs46q81kc9FXCjqRRpvmadk2Y+edcowVNBDRn0r/bV2eoLKOhlnBgAAAABQFASOwHYoqxqOqfsyKPxv34gaiZn6htZJzxb1mo//4+HqwP8lkseIXN3apHo9tfLNe//4g9zkD0cTsawajnqSmBnP2pzNGziurz+d/J8L/7ZP7PDsvvnevEPTbEaRAkdL9hfpFrGddTrT80hb5Zcfv+l4VySMjUdOvP+FGARGX9Me8D1P4PijR1SQtL7lGDhmbWWdJXAEAAAAAGwbAkdgO5RR4ChyvcWl/7bnH2f/t4WHIiJUNRwDuaOITC1++F/+8W9diW5LbmgLHHWpZKRDRIT/fkvbnbWGo2yUbfQaqW/4x66VDQNHXfo/+he0QxtdTL6YogSOT25nWjereosne77X38qWjg99Jho4f3B7QdVwVK2blecIHPW2242ffivqkW4QOK4MfqS91WC8pYeMeneTBI4AAAAAgG1D4AhshzILHNfXJmXtxQ//u2jju/SNiAL/cHQhoLeqvrU4MCVWWl9PBd75x1nZD+P/eKiGpbb24Rj574v3xXDSyYddYmF24Li+8j9kv41//u9LkW8W/yxfy/3kDRzn+xe/+WY59n1q5n5CnG3ZBI7nPuq+0t198f1jst5iZriV76+Jiodvf+q5J1tV3/vm9vDK+sLQ378eDA1Pzs2M9OhR4IKxZsN7n2lvRQZvfz1i9OG4YeCor3+vT44Dc6IjJI/69NsO7a0j73xya/Be6JF9P4m7rdoZnnB3B0KRQPc58VoNqE3gCAAAAADYNgSOwHYot8BxfT31nz3/m0j9bk2tra8/jfXM6y2jxbTnH/9D5oUaPYs8+78vTqoFtsBxIHtsa401cFxfn1r6c2bU6T3/+PM3er3I/IHjN5bT2Df/jTi3F1bIwNGYGt45/1ngB0tPjD/f+9QcSPrIiY/uJNYXAh9Zlpw3ajVa1zzdGlCjVG8cOB577/23ZcRpNM3W/XDrPTXgdWa068x+Vh5dEzmjfNe6FYEjAAAAAGDbEDgC26EMAsdneyrGhl58+nwjQ6fkVql/qlkHa+nn2206+QKn4WD7yvzpL2JA6hXrGaflkkxEmKGt+ctmOqa0hI/arnL3o0nn30/6qfPRAQAAAADYDgSOwHZ4KQLHCvNSl7m1tiMAAAAAAC8XAkdgOxA4br+XuswXAp+83XT248CcmgcAAAAA4OVB4AhsBwLH7UeZAwAAAABQEgSOwHYg/Np+lDkAAAAAACVB4AhsB8Kv7UeZAwAAAABQEgSOwHYg/Np+lDkAAAAAACVB4AhsB8Kv7UeZAwAAAABQEgSOwHYg/Np+lDkAAAAAACVB4AhsB8Kv7UeZAwAAAABQEgSOwHYg/AIAAAAAADsEgSOwHQgcAQAAAADADkHgCGwHAkcAAAAAALBDEDgC24HAEQAAAAAA7BAEjsB2IHAEAAAAAAA7BIEjsB0IHAEAAAD8/9uxYxQEgiiIgrcQjLz/KVeDDUUE5SF2FZ1O8mGSBzBCcISC4AgAAACMEByhIDgCAAAAIwRHKAiOAAAAwAjBEQqCIwAAADBCcISC4AgAAACMEByhIDgCAAAAIwRHKHwlOF6uN3t/59UAAACAluAIBcGx33k1AAAAoCU4QuHD4Ph4bmZmZmZmZmb2OzubxTOCIxRe/0MAAACAvyE4QkFwBAAAAEYIjlAQHAEAAIARgiMUBEcAAABghOAIBcERAAAAGCE4QkFwBAAAAEYIjlAQHAEAAIARgiMUBEcAAABghOAIBcERAAAAGCE4QkFwBAAAAEYIjlAQHAEAAIARgiMUBEcAAABghOAIBcERAAAAGCE4QkFwBAAAAEYIjlAQHAEAAIARgiMUBEcAAABghOAIBcERAAAAGCE4QkFwBAAAACYcxx18wFwp34sG7QAAAABJRU5ErkJggg=="
```

Comment:
I actually used our color for the boxes. I can make two new images and update them later. It's ok if this PR merges with the current image.

Reply:
I chose a similar green from MS paint lol